packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-card-front.tsx

```diff
@@ -1,48 +1,77 @@
+import React from 'react';
+import classNames from 'classnames';
 import { Text } from '@deriv/components';
 import { Localize } from '@deriv/translations';
 import { TJurisdictionCardFrontProps } from 'Containers/props.types';
-import React from 'react';
 import JurisdictionCardSection from './jurisdiction-card-section';
 
 const JurisdictionCardFront = ({
     card_classname,
     toggleCardFlip,
     card_values,
     card_data,
+    is_card_selected,
+    disabled,
 }: TJurisdictionCardFrontProps) => (
-    <div className={`${card_classname}__card-content-container`}>
-        {card_values.is_over_header_available ? (
+    <div
+        className={classNames(
+            card_classname,
+            'cfd-card-front',
+            {
+                [`${card_classname}--selected selected-card`]: is_card_selected,
```

Comment:
why do we add 2 classes here?
${card_classname}--selected and selected-card

Reply:
@amina-deriv css :not selector was not working with the scss based names. so i wrote a new class for not selector alone.
&:not(.selected-card):hover {
Is there a way to resolve already existing classname &--selected inside :not operator like &:not(.&--selected):hover {?